video_xblock/static/js/studio-edit/transcripts-manual-upload.js

```diff
@@ -240,11 +240,14 @@ function createTranscriptBlock(langCode, langLabel, transcriptsValue, downloadTr
  */
 function clickUploader(event, $fileUploader) {
     'use strict';
+    var handoutsAllowedFileTypes = '.png, .jpeg, .jpg, .gif, .tiff';
```

Comment:
http://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/course_assets/course_files.html#filter-listed-files